memory/ndctl.py

```diff
@@ -431,20 +444,22 @@ def test_multiple_ns_multiple_region(self):
         self.disable_namespace()
         self.destroy_namespace()
         for val in regions:
+            ns_size = None
             region = self.get_json_val(val, 'dev')
             self.log.info("Using %s for muliple namespaces", region)
             self.log.info("Creating %s namespaces", self.cnt)
             if not self.size:
-                self.size = self.get_json_val(self.get_json(
```

Comment:
why variable name change requires is it for readability ?

Reply:
It is being using inside the for loop as a check. As stated in commit message it has to be changed to a local variable to have a proper check throughout the loop.